### PR TITLE
Llm stratify

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
Add llm toolbar to stratify operator for code generation

Note:
- Not all prompts work, however this template of `show me code to stratify model abc by region named xyz and zzz` had worked more consistently
- The code generated is not precisely correct, it should be "model = stratify(..." instead of "stratified_model = stratify(...". However this is outside of the scope of this PR to address.

image

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue [hunk 1]
```diff
@@ -41,8 +41,14 @@
 			</tera-drilldown-section>
 		</div>
 		<div :tabName="StratifyTabs.Notebook">
-			<tera-drilldown-section>
-				<p class="mt-3 ml-4">Code Editor - Python</p>
+			<tera-drilldown-section class="notebook-section">
+				<tera-notebook-jupyter-input
```
Comment (Tom-Szendrey) [inline packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:45]: I am confused why you do not need `Suspense`

Either its not required or I am unable to click fast enough to send something before the kernelManager is built so it doesnt appear to be a problem.

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue [hunk 2]
```diff
@@ -185,6 +192,7 @@ const outputs = computed(() => {
 	return [];
 });
 
+const kernelReady = ref(false);
```
Comment (Tom-Szendrey) [inline packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:195]: I see that youre setting this value, but I do see where you're using it?

**Review by Tom-Szendrey — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue [hunk 3]
```diff
@@ -483,6 +497,16 @@ onUnmounted(() => {
...
+.notebook-section:deep(main .notebook-toolbar),
+.notebook-section:deep(main .ai-assistant) {
+	padding-left: var(--gap-medium);
+}
+
+.notebook-section:deep(main) {
+	gap: var(--gap-small);
+	position: relative;
+}
+
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:509]: Maybe write a fix me so someone can fix the css rules in the component so its more plug and play

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue (hunk 3 above)
Comment (mwdchang) [reply packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:507]: will follow up offline.

**Pull request merged**